### ISSUE TITLE
Question about duration predictor

### ISSUE THREAD

**Issue by Jackson-Kang** [ming024/FastSpeech2#43]
Hello @ming024  ,

First of all, thank you for sharing your awesome work.

I leave this question due to my curiosity about duration predictor improvements.
A few months ago, evaluation performance of duration predictor seemed to be not good due to overfitting. 
(At that time, train error was less than 0.01, but eval error was 0.5~0.6.)

But, now it has been drastically improved (in the past, eval error was 0.5~0.6, but now 0.08 ~ 0.1).
If you don't mind, could you tell me what was the problem of your previous version of duration predictor?

Always appreciate,

**Comment by ming024** [comment]
@Jackson-Kang I think it is possibly because that the pitch and energy values are normalized in the current implementation so the magnitudes of the losses are on the same scale, which makes the optimization more balanced.

**Comment by Jackson-Kang** [comment]
I had tested a few months ago. 
However, the normalization of pitch and energy does not have severe impact on the duration predictor overfitting..

Anyway, thank you for answering my question.
Sincerely,
